gef-ui/server/controllers/activities-controller/index.js

```diff
@@ -4,25 +4,35 @@ const { getApplication, getUserDetails } = require('../../services/api');
 
 // maps portalActivities array to create array in correct format for mojTimeline
 const mapPortalActivities = (portalActivities) =>
-  portalActivities.map((portalActivity) => ({
-    label: {
-      text: portalActivity.label,
-    },
-    text: portalActivity.text,
-    datetime: {
-      timestamp: fromUnixTime(new Date(portalActivity.timestamp)),
-      type: 'datetime',
-    },
-    byline: {
-      text: `${portalActivity.author.firstName} ${portalActivity.author.lastName}`,
-    },
-    html: portalActivity.html,
-    facilityType: portalActivity.facilityType,
-    ukefFacilityId: portalActivity.ukefFacilityId,
-    facilityId: portalActivity.facilityId,
-    maker: portalActivity.maker,
-    checker: portalActivity.checker,
-  }));
+  portalActivities.map(({ label, text, timestamp, author, html, facilityType, ukefFacilityId, facilityId, maker, checker }) => {
+    let bylineText = author.firstName;
+
+    if (author.lastName) {
+      bylineText += ` ${author.lastName}`;
```

Comment:
For the new "deal cancelled" activity, the name is "UKEF", so there is no last name.